<launch>

    <include file="$(find kinect_challenge)/launch/view_robot.launch" />
  
    <node type="MSBMIOSvs.py" name = "MSBMIOSvs" pkg="kinect_challenge" />
    <node type="Navigator.py"    name = "Navigator"    pkg="kinect_challenge" >
  
    <param name="MapPath" type="str" value="$(find kinect_challenge)/maps" />
    <param name="WaypointFile" type="str" value="waypoints.wp" />
    <param name="WaypointTour" type="str" value="tour.wpt" />
   
    
  </node>
</launch>
